hazardous/_survival_boost.py

```diff
@@ -505,13 +515,29 @@ def predict_cumulative_incidence(self, X, times=None):
             predictions_at_t = self.estimator_.predict_proba(X_with_time)
             predictions_at_all_times.append(predictions_at_t)
 
-        predicted_curves = np.array(predictions_at_all_times).swapaxes(2, 0)
-
-        return predicted_curves
+        predicted_curves = np.array(predictions_at_all_times)
+        # roll axis to get a shape (n_samples, n_events + 1, n_times)
+        return np.rollaxis(predicted_curves, 1).swapaxes(1, 2)
```

Comment:
Might be easier to follow with a single call to `np.transpose(predicted_curves, axes=(1, 2, 0))` instead.